CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 1.15.0
+
+* Re-export a number of platform library classes from `dart:collection`
+  in anticipation of moving the classes to this package for Dart 2.0.
```

Comment:
List the classes that are being re-exported, so users of those classes can search the changelog for a class name to see which version of the package first exposed that class.